subsys/settings/Kconfig

```
@@ -90,6 +91,19 @@ config SETTINGS_CUSTOM
 	help
 	  Use a custom settings storage back-end.
 
+config SETTINGS_ITS
+	bool "Internal Trusted Storage (ITS)"
+	help
+	  Enables ITS settings storage back-end. Enables an app to store sensitive
+	  settings information to PSA Internal Trusted Storage (ITS). This is useful
+	  for storing sensitive settings information such as keys, passwords, or other
+	  secrets. This also enables code running from non-secure partition to make use
+	  of the settings subsystem (secure code can use this backend as well). ITS may
```

```suggestion
	  secrets. 
	  This also enables code running from non-secure partition to make use
	  of the settings subsystem (secure code can use this backend as well). ITS may
```
This sentence belongs to the ITS Kconfig description IMO.

---

Okay I added more to the description.